src/test/java/uk/gov/hmcts/reform/coh/controller/EventForwardingControllerTest.java

```diff
@@ -1,64 +1,141 @@
 
-        eventRegistrationRequest = (EventRegistrationRequest) JsonUtils.toObjectFromTestName("event_forwarding_register/valid_event_register_no_retries", EventRegistrationRequest.class);
+        String json = JsonUtils.getJsonInput("event_forwarding_register/valid_event_register");
 
-        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT)
-                .contentType(MediaType.APPLICATION_JSON).content(JsonUtils.toJson(eventRegistrationRequest)))
-                .andExpect(status().is2xxSuccessful());
+        given(sessionEventForwardingRegisterService.retrieveEventForwardingRegister(
+                any(SessionEventForwardingRegister.class)))
+                .willReturn(Optional.of(new SessionEventForwardingRegister()));
 
-        assertEquals(DEFAULT_RETRIES, eventRegistrationRequest.getMaxRetries());
+        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT+"/register")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(json))
+                .andExpect(status().is4xxClientError());
+
+    }
+
+    @Test
+    public void testCreateEventForwardRegisterMissingJurisdiction() throws Exception {
+
+        String json = JsonUtils.getJsonInput("event_forwarding_register/valid_event_register");
 
+        given(sessionEventTypeService.retrieveEventType(any(String.class)))
+                .willReturn(Optional.empty());
+
+        given(sessionEventForwardingRegisterService.retrieveEventForwardingRegister(
+                any(SessionEventForwardingRegister.class)))
+                .willReturn(Optional.of(sessionEventForwardingRegister));
+
+        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT+"/register")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(json))
+                .andExpect(status().is4xxClientError());
     }
 
     @Test
-    public void testIsActiveIsBoolean() throws Exception {
-        eventRegistrationRequest = (EventRegistrationRequest) JsonUtils.toObjectFromTestName("event_forwarding_register/event_register_invalid_active", EventRegistrationRequest.class);
+    public void testCreateEventForwardRegisterMissingEventType() throws Exception {
+
+        String json = JsonUtils.getJsonInput("event_forwarding_register/valid_event_register");
+
+        given(jurisdictionService.getJurisdictionWithName(any(String.class)))
+                .willReturn(Optional.empty());
+
+        given(sessionEventForwardingRegisterService.retrieveEventForwardingRegister(
+                any(SessionEventForwardingRegister.class)))
+                .willReturn(Optional.of(sessionEventForwardingRegister));
 
-        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT)
-                .contentType(MediaType.APPLICATION_JSON).content(JsonUtils.toJson(eventRegistrationRequest)))
+        mockMvc.perform(MockMvcRequestBuilders.post(ENDPOINT+"/register")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(json))
```

Comment:
Explicitly check the HTTP status code just because you're using spring validation and need to be sure that it returns the correct value